source/_posts/2020-09-18-mac-companion.markdown

```diff
@@ -0,0 +1,60 @@
+
+TODO: ADD VIDEO EMBED HERE
+
+One huge advantage of running on a Mac compared to a mobile device is the much larger battery. This means we are not constrained by battery saving measures and can address one of the most common gripes with the iOS app, update intervals. On a Mac entity updates are triggered immediately when something changes. You will see this reported by the `sensor.DEVICE_NAME_last_update_trigger` reporting `Signaled`.
+
+### Notifications
+
+Just like the iOS app, you can send notifications to your Mac with services like `notify.mobile_app_DEVICE_NAME`. One small difference is that [critical notifications](https://companion.home-assistant.io/docs/notifications/critical-notifications) are not yet available for the Mac app. However, all our other notifications features like actionable notifications work on the Mac app. To see what is possible, take a look at [the docs](https://companion.home-assistant.io/).
+
+### Interface
+
+Your Mac isn't a mobile so you shouldn't be forced to use an interface designed for a mobile. In the Mac app the App Configuration page has been removed from Home Assistant's sidebar and instead the configuration options and preferences are in the Home Assistant menu on the menu bar right where you'd expect to find them for any other app and all the standard shortcuts work too (just hit `cmd + ,` for preferences). Here you will also find an option to manually send an update to Home Assistant and you can even open extra Home Assistant windows! Finally, on the menu bar you can also find a new Actions menu where you can see all your [actions](https://companion.home-assistant.io/docs/core/actions) and fire them.
+
+### Big Sur Features
+
+Home Assistant Companion for Mac already supports widgets in Big Sur. Right now we have an Actions Widget where you can have up to eight actions. You can also create multiple widgets with different sets of actions. If you have an idea for other widgets you'd like to see, pop over to [the community forums and let us know](https://community.home-assistant.io/t/what-kind-of-ios-14-widgets-would-you-like-to-see/211112/14).
+
+<p class='img'>
+<img src='/images/blog/2020-09-18-mac-companion/actions-widget.png' alt='Screenshot of a large Home Assistant Actions widget in Big Sur'></a>
+Screenshot of a large Home Assistant Actions widget in Big Sur.
+</p>
+
+### Documentation and Support
+
+We are updating the [Companion App docs](https://companion.home-assistant.io) with details for the Mac app. You can also pop over to the [Discord channel](https://discord.com/login?redirect_to=%2Fchannels%2F330944238910963714%2F551871772484698112). If you find a bug or have an idea for a feature please open an issue over on the [GitHub repository](https://github.com/home-assistant/iOS/issues/new/choose).
```

Comment:
Invites can expire, should we write something like

>Join us on #ios on [Discord](https://www.home-assistant.io/join-chat/)

Reply:
I don't think that is an invite link, I believe that's the actual channel address. It matches up with the URL seen in the browser version. I took it from the companion docs where it has been for ageeeeees